.eslintignore

```diff
@@ -1,4 +1,5 @@
 coverage/
+examples/
```

Comment:
It's going to throw a lot of errors about globals and arrow functions and the like that I don't think we necessarily need to follow in these examples since we don't run them through babel.